x-pack/plugins/cases/common/api/cases/user_actions.ts

```diff
@@ -51,6 +50,8 @@ const CaseUserActionResponseRT = rt.intersection([
     action_id: rt.string,
     case_id: rt.string,
     comment_id: rt.union([rt.string, rt.null]),
+    new_val_connector_id: rt.union([rt.string, rt.null]),
```

Comment:
These will be populated with the connector ids extracted from the `newValue` and `oldValue` fields.